src/io/opendss.jl

```diff
@@ -1,6 +1,14 @@
 # OpenDSS parser
 
 
+"parses 'source_id' into Dict containing the 'type', 'name' and active 'phases' of the OpenDSS component"
+function parse_source_id(source_id::String)
+    component_type, component_name, active_phases = split(source_id, '.', limit=3)
+    nodes = [contains(active_phases, "$n") for n in 1:3]
+    return Dict("type"=>component_type, "name"=>component_name, "phases"=>nodes)
```

Comment:
This may be the exception to our Dict standard, a tuple or struct may be more appropriate.  Can you think of any case where one would want to extend the the fields these three fields?  If not, probably a struct is a better fit.